pages/team.js

```diff
@@ -547,11 +279,9 @@ When not busy juggling different tasks he takes up, he enjoys tinkering & buildi
 
 export const getServerSideProps = async () => {
```

Comment:
@DaInfLoop all you need to do is use static probs instead of server side props and no need to cache i suppose

Reply:
just make sure to validate it in the background every so and so minute